assessment.md

```diff
@@ -13,81 +13,512 @@
+    * A = Enabling Travis, and passing a majority of Travis checks
+    * G = Enabling Travis, and passing all of the Travis checks
+    * E = Enabling Travis, passing all checks, and assisting others who are
+      struggling or improving issues with Travis if found
+  * Foundations of software engineering practices
+    * N = Working separate from a team with no communication
+    * I = Working with a team a small amount of the time with no communication
+    * A = Working well with a team half of the time with little communication
+    * G = Working well with a team most of the time with clear communication
+    * E = Working well with a team all time with clear and constructive
+      communication
 * Understanding and avoiding red flags
+  * N = Ignoring all red flags
+  * I = Would be identifying red flags, but not fixing them
+  * A = Identifying and fixing a red flag
+  * G = Identifying, understanding and fixing a red flag to prevent future bugs
+  * E = Identifying, understanding, and fixing all red flags
 * Understanding and adopting best practices
+  * N = A severe lack of understanding and refusal to adopt best practices
+  * I = Slight understanding and occasional use and identification of the
+    best practices
+  * A = An average amount of understanding and general identification and use of
+      best practices
+  * G = Above average demonstrations of understanding and the ability to find and
+    utilize best practices
+  * E = Full understanding of concepts, would also include the ability to determine
+    which practices are better than others and then applying those practices
 
 ### Group
 
 * Attendance
+  * N = Missing five or more lab sessions
+  * I = Missing four lab sessions
+  * A = Missing three lab sessions
+  * G = Missing two lab sessions
+  * E = Missing one or less lab sessions
 * Deadlines
+  * N = If an assignment is turned in 5 or more days past the deadline
+  * I = If an assignment is turned in 1-4 days past the deadline
+  * A = If an assignment is turned in on the day of, or before the deadline
+  * G = If an assignment is turned in on the day of, or before the deadline
+  * E = If an assignment is turned in on the day of, or before the deadline
 * Communication
+  * N = Never check appropriate slack channels and never communicate with
+   teammates
+  * I = Rarely check appropriate slack channels and rarely communicate with
+    teammates
+  * A = Sometimes check appropriate slack channels and sometimes communicate
+    with teammates
+  * G = Regularly check appropriate slack channels and regularly communicate
+    with teammates
+  * E = Always check appropriate slack channels and always communicate
+    effectively with teammates
 * Participation
+  * N = Never participate in group conversation or activity
+  * I = Rarely participate in group conversation or activity
+  * A = Sometimes participate in group conversation or activity
+  * G = Regularly participate in group conversation or activity
```

Comment:
Is there a difference between regularly and always? This seems confusing.

Reply:
I agree here. Perhaps using a time amount expected would be better?